src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.*;
```

Comment:
Avoid using * to import all classes from the package. Import the classes individually